Formula/jsonnet-bundler.rb

```diff
@@ -0,0 +1,29 @@
+class JsonnetBundler < Formula
+  desc "Package manager for Jsonnet"
+  homepage "https://github.com/jsonnet-bundler/jsonnet-bundler"
+  url "https://github.com/jsonnet-bundler/jsonnet-bundler.git",
+    :tag      => "v0.4.0",
+    :revision => "447344d5a038562d320a3f0dca052611ade29280"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "static"
+    bin.install "_output/jb"
+  end
+
+  test do
+    run_output = shell_output("#{bin}/jb 2>&1")
+    assert_match "A jsonnet package manager", run_output
+
+    cd testpath.realpath do
+      system bin/"jb", "init"
+      File.exist?("jsonnetfile.json")
+
+      system bin/"jb", "install", "https://github.com/grafana/grafonnet-lib"
+      File.directory?("vendor")
+      (File.file?("jsonnetfile.lock.json") && !File.zero?("jsonnetfile.lock.json"))
+    end
```

Comment:
```suggestion
    system bin/"jb", "init"
    assert_predicate testpath/"jsonnetfile.json", :exist?
```

Reply:
I disagree with this change, although not with the `:exist?` part. @SMillerDev indicated that the tests should give an indication of the basic workflow. The above code is the basic workflow, i.e.

- Init the repo
- Install a Jsonnet library
- Test if the lock file is successfully produced.

Reply:
Fixed.